tests/test_data/credentials.py

```diff
@@ -1,3 +1,3 @@
 class Credentials:
     login_email = 'dytaza@mailto.plus'
-    login_passw = 'qwerty654321'
\ No newline at end of file
+    login_passw = 'qwerty654321'
```

Comment:
please don't keep passwords and access keys from real accounts inside code.
The repository is public, so everybody can stole them.
I suggest to pass the sensitive data like environment variables and resolve them on the fly running tests.

Reply:
It's a temporary account, I don't value them.